V1OOP/test/practium_3a/VoetbalclubTest.java

```diff
@@ -6,6 +6,12 @@
 
 class VoetbalclubTest {
 
+    @Test
+    public void testGetClubnaam(){
+        Voetbalclub twente = new Voetbalclub("FC Twente");
```

Comment:
Je kan ook "FC Twente" in een variable zetten zodat hij vast staat als je een van de 2 vervangt.